src/storage/src/client.rs

```diff
@@ -737,13 +737,13 @@ impl ReadObjectResponse {
     ///     .send()
     ///     .await?;
     ///
-    /// while let Some(next) = resp.next().await? {
-    ///     println!("next={next:?}");
+    /// while let Some(next) = resp.next().await {
+    ///     println!("next={:?}", next?);
```

Comment:
Consider writing this as:

```suggestion
    /// while let Some(next) = resp.next().await.transpose()? {
    ///     println!("next={next:?}");
```